requirements-dev.txt

```
@@ -1,4 +1,4 @@
-git+https://github.com/PennyLaneAI/pennylane.git@master
+git+https://github.com/PennyLaneAI/pennylane.git@v0.38.0-rc0
```

We may want to update that.
```suggestion
git+https://github.com/PennyLaneAI/pennylane.git@master
```